pages/embedded-portal.tsx

```diff
@@ -110,7 +109,7 @@ const EmbeddedPortal: NextPageWithLayout<Props> = ({
 
           res.actions.forEach((a) => {
             toast.success(a.description, {
-              id: DateTime.now().toISO(),
+              id: DateTime.now().toISO() as string,
```

Comment:
why's this needed?

Reply:
Getting this error. Found a better way to deal with it vs using "as string"

![CleanShot 2023-06-13 at 11 14 24](https://github.com/FlatFilers/HCMShow/assets/12041898/d47cedfe-f767-4134-9bd8-e3746be11bed)